docs/ContactUs.adoc

```diff
@@ -2,6 +2,6 @@
 :site-section: ContactUs
 :stylesDir: stylesheets
 
-* *Bug reports, Suggestions* : Post in our https://github.com/se-edu/addressbook-level4/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
+* *Bug reports, Suggestions* : Post in our https://github.com/cs2103-ay1819s2-t12-3/main/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
 * *Contributing* : We welcome pull requests. Follow the process described https://github.com/oss-generic/process[here]
-* *Email us* : You can also reach us at `damith [at] comp.nus.edu.sg`
+* *Email us* : You can also reach us at `e0191399 [at] comp.nus.edu.sg`
```

Comment:
I am not sure if you really have the email domain name `comp.nus.edu.sg`. It might actually be `e0191399 [at] u.nus.edu`

Reply:
This has resolved in commit 2106eca